libs/packages/components/src/lib/side-navigation/side-navigation.component.ts

```diff
@@ -141,20 +154,19 @@ export class SdsSideNavigationComponent {
    * event for event based
    */
   @Output()
-  linkEvent = new EventEmitter<INavigationLink>();
-
+  navigate = new EventEmitter<INavigationLink>();
   /**
    * Link clicked and emits the link data into an event
-   * @param link 
+   * @param link
    */
-  linkClickEvent(link: INavigationLink) {
-    this.linkEvent.emit(link);
+  navigation(link: INavigationLink) {
+    this.navigate.emit(link);
     return false;
   }
```

Comment:
Revert this change back to being named linkEvent, this will prevent a breaking change.